GetDKAN/dkan2
# Demo search config

##QA 

Depends on https://github.com/GetDKAN/dkan-tools/pull/166

- dkan-tools `git checkout V8-basics`
- `dktl make --branch=demo-search-config`
- `dktl install --demo`
- Confirm dkan index has title, description, keywords, and theme
